playbooks/lvm.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: lvm
```

Comment:
I think we'd probably want to use group vars or host vars to define the expected LVM bits, like `compute` group gets the `/var/lib/nova/instances` path, while others would get.... something.